pypeit/pypeit.py

```diff
@@ -251,27 +252,36 @@ def get_std_outfile(self, standard_frames):
             msgs.error('Could not find standard file: {0}'.format(std_outfile))
         return std_outfile
 
-    def calib_all(self):
+    def calib_all(self, run=True):
```

Comment:
Can we just make it a default to write out this file always when PypeIt is run. In that we can remove the run keyword and we can also get rid of the script. Then by construction anyone that runs pypeit will have a file generated which indicates the calibration-file association. I don't see why we would not want to *always* write out that file, and this also results in less code to maintain and test (i.e. the script).

Reply:
yes, that is easy enough.  will do

Reply:
ok, a file with a `.calib_ids` extension is now generated.

But, I am keeping the script, in large part in case
the calibration generation fails part way through 
but produces items that they wish to decipher.

Or in case they modify the PypeIt file and want to 
know what that will now bring.